api/api-doc/storage_service.json

```diff
@@ -2844,7 +2844,7 @@
                "nickname":"repair_tablet",
                "method":"POST",
                "summary":"Repair a tablet",
-               "type":"void",
+               "type":"string",
```

Comment:
Doesn't it return JSON actually?
Let's look around and see what other API which return JSON object declare in their definition.

Reply:
I checked if a json type is used before sending the PR and I did not find anything.  It turned out we need to define a custom type. I did that in v2.